library/StandardLibrary/test/NameSymb.hs

```diff
@@ -1,11 +1,15 @@
 module NameSymb where
 
+--------------------------------------------------------------------------------
+
 import Juvix.Library
 import qualified Juvix.Library.NameSymbol as NameSymbol
 import qualified Test.Tasty as T
 import qualified Test.Tasty.HUnit as T
 import qualified Test.Tasty.QuickCheck as T
 
+--------------------------------------------------------------------------------
+
```

Comment:
Sections around imports aren't really needed as they have to come after the module with nothing in between due to how haskell handles modules

Reply:
This is a style issue. It's not a need, of course. So I can keep it as it was, no big deal. But, maybe,  the style guide we have should include more discussion on section, naming, comments, docs, etc. As the whole codebase is not consistent. But it's not a priority now, so let us keep it for later.